std/container/rbtree.d

```diff
@@ -1368,6 +1399,18 @@ assert(equal(rbt[], [5]));
         return removeKey(toRemove[]);
     }
 
+    ///
+    unittest
```

Comment:
I just realized, this is inside the template itself. Unfortunately, the way unit tests work, this will be run on any instantiation of `RedBlackTree` during unit tests (including user unit tests!), which is why I have the whole `doUnittest` enum.

I'm thinking in this case, we should leave the example in ddoc form.